workbench-core/example/infrastructure/integration-tests/tests/users/createUser.test.ts

```diff
@@ -20,7 +20,7 @@ describe('userManagement create user integration test', () => {
     user = {
       firstName: 'Test',
       lastName: 'User',
-      email: 'fakeemail@notanemail.com'
+      email: 'create@user.com'
```

Comment:
If this is actually calling into Cognito, be careful. You don't want invalid emails going out to `create@user.com` and damaging the sender's reputation. Consider using a valid/unmonitored email address such as `success+create-user@simulator.amazonses.com`

Reply:
fixed!